### DISCUSSION TITLE
Бесконечная ловкость

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2_classic#70] [General]
А нужно ли исправлять возможность получения бесконечных зелий ловкости на Ирдорате?

**Comment by Kor-Angar** [comment]
> А нужно ли исправлять возможность получения бесконечных зелий ловкости на Ирдорате?

"бесконечная ловкость" видимо нужна что бы взять лук который есть в сундуке возле дракона. надо будет этот момент перепроверить.

возможно это сделано на тот случай если герой приедит на Ирдорат без лука и ловкости. тогда он не сможет пройти вглубь острова.

**Comment by dosinabox** [comment]
Можно давать ГГ столько ловкости, сколько ему необходимо для этого лука в сундуке.
А сколько на Ирдорате бесплатных стрел (не у торговцев с корабля)?

**Comment by Kor-Angar** [comment]
Надо проверить будет запустить игру и сбегать. Сегодня могу посомтреть.Там в одном сундуке лук, а в другом стрелы. Сундук взлома не требуют. Стрел по моему больше 10.

**Comment by Kor-Angar** [comment]
кажется вот этот
![image](https://user-images.githubusercontent.com/49514764/86582201-f03a5180-bf89-11ea-8e2f-77258773d9ae.png)

в НВ вроде 11. в Готике 2 классик надо проверять
![image](https://user-images.githubusercontent.com/49514764/86582317-1fe95980-bf8a-11ea-8469-fe144cc20ad1.png)

**Comment by Kor-Angar** [comment]
Да. там Лук наёмника 20 ущерба, надо 20 ловкости.
![image](https://user-images.githubusercontent.com/49514764/86659225-5c50a000-bff2-11ea-8b7e-096028f692d5.png)
стрел 5+11=16

**Comment by Kor-Angar** [comment]
> А нужно ли исправлять возможность получения бесконечных зелий ловкости на Ирдорате?

а какой там скрипт? при каких условиях срабатывает? если у героя ловкости меньше 20?

**Comment by dosinabox** [comment]
Если меньше 15. Что не имеет смысла, потому что до 20 для бесплатного лука из сундука может всё равно не хватить. Поэтому нужно считать разницу между 20 и текущей ловкостью и вставлять столько зелий, сколько нужно. То же касается и Г2НВ.

**Comment by dosinabox** [comment]
Или можно просто вставить где-то на острове лук попроще. Чтобы его мог взять любой. На торговцев лучше не полагаться, потому что их может и не быть в команде. Или погибнут при атаке орков.

**Comment by Kor-Angar** [comment]
Можно вместо зелья ловкости вставить самый простой лук

**Comment by Kor-Angar** [comment]
у тебя миры Ирдората в классике и НВ одинаковые? :-)

**Comment by dosinabox** [comment]
с версии 1.2 - да

**Reply by Kor-Angar** [reply]
> кстати, в классике теперь тоже на корабль будет вставляться лук

**Comment by dosinabox** [comment]
кстати, в классике теперь тоже на корабль будет вставляться лук

**Comment by Kor-Angar** [comment]
вообщем в MDK прямо в примечаниях было написано что вставка эликсира нужна для лука.
